Silicon/CommonSocPkg/Library/HeciLib/HeciCore.c

```diff
@@ -988,3 +989,38 @@ HeciServiceResetInterface (
 {
   return HeciResetInterface((HECI_DEVICE)HeciDev);
 }
+
+/**
+  Check for Manufacturing Mode
+
+  @param[out]  MeManuMode         Manufacturing mode/production mode.
+
+  @retval EFI_UNSUPPORTED         Current CSME mode doesn't support this function
+  @retval EFI_SUCCESS             Command succeeded
+
+**/
+EFI_STATUS
+EFIAPI
+HeciGetManufactureMode (
+  OUT UINT16 *MeManuMode
+  )
+{
+
+  HECI_FW_STS6_REGISTER MeFirmwareStatus;
+
+  MeFirmwareStatus.ul = HeciPciRead32 (HECI_FUNCTION_NUMBER, R_ME_HFS_6);

```

Comment:
There is no HeciPciRead32() API in common library. Please refer to HeciGetMeMode() for your implementation.

Reply:
Done